plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -184,32 +184,24 @@ public static void installAllAppEngineRuntimes(IFacetedProject project, IProgres
     }
   }
 
-  public static org.eclipse.wst.server.core.IRuntime createAppEngineServerRuntime(IProgressMonitor monitor)
-      throws CoreException {
+  public static org.eclipse.wst.server.core.IRuntime createAppEngineServerRuntime(
+      IProgressMonitor monitor) throws CoreException {
     IRuntimeType appEngineRuntimeType =
         ServerCore.findRuntimeType(AppEngineStandardFacet.DEFAULT_RUNTIME_ID);
     if (appEngineRuntimeType == null) {
-      throw new NullPointerException("Could not find " + AppEngineStandardFacet.DEFAULT_RUNTIME_NAME + " runtime type");
+      throw new NullPointerException(
+          "Could not find " + AppEngineStandardFacet.DEFAULT_RUNTIME_NAME + " runtime type");
     }
 
     IRuntimeWorkingCopy appEngineRuntimeWorkingCopy
```

Comment:
where does this runtime get the Cloud SDK from now? Looking at this PR, it seems to me I was wrong earlier the Cloud SDK is indeed required at this stage.